tests/e2e/tidbcluster/serial.go

```diff
@@ -747,42 +748,75 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			framework.ExpectNoError(err, "create pdapi error")
 			defer cancel()
 			var firstScaleTimestamp int64
-			err = wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
+			var readyToScaleTimestamp int64
+			err = wait.Poll(10*time.Second, 5*time.Minute, func() (done bool, err error) {
+				tac, err = cli.PingcapV1alpha1().TidbClusterAutoScalers(ns).Get(tac.Name, metav1.GetOptions{})
+				if err != nil {
+					return false, nil
+				}
+				if tac.Annotations == nil || len(tac.Annotations) < 1 {
+					framework.Logf("tac haven't marked any annotation")
+					return false, nil
+				}
+				t, ok := tac.Annotations[label.AnnTiKVReadyToScaleTimestamp]
+				if !ok {
+					framework.Logf("tac has no tikv.tidb.pingcap.com/ready-to-scale-timestamp annotation")
+					return false, nil
+				}
+				readyToScaleTimestamp, err = strconv.ParseInt(t, 10, 64)
+				if err != nil {
+					return false, err
+				}
+				if tac.Status.TiKV.Phase != v1alpha1.ReadyToScaleOutAutoScalerPhase {
+					framework.Logf("tac dont' have the right ReadyToScale phase, expect: %s, got %s", v1alpha1.ReadyToScaleOutAutoScalerPhase, tac.Status.TiKV.Phase)
```

Comment:
use log instead of return error because we only reset phase but not timestamp, so in second scale, the timestamp is there, but the tac haven’t change phase.